public/app/features/alerting/unified/hooks/useExternalAMSelector.test.ts

```diff
@@ -126,4 +126,26 @@ describe('useExternalAmSelector', () => {
       },
```

Comment:
The other tests are failing because the urls are not valid, they need to be updated with a valid url to not throw an exception

https://developer.mozilla.org/en-US/docs/Web/API/URL/URL